frontend/src/pages/user/UserList.tsx

```diff
@@ -136,15 +158,15 @@ const UserList: React.FC = () => {
       },
     },
     {
-      id: 'role',
+      id: 'roles',
       header: t('user:labels.tableColumnRole'),
-      minWidth: 200,
+      minWidth: 240,
       editConfig: {
         editingCell: (item: IUserTableItem, { setValue, currentValue }: any) =>
           renderEditRoleCell(item, setValue, currentValue),
       },
-      cell: (e: { role: string }) => {
-        return getRoleName(e.role);
+      cell: (e: { roles: IUserRole[] }) => {
```

Comment:
Update the **e** type to `IUserTableItem`

Reply:
Modified.